modules/platforms/dotnet/Apache.Ignite.Core.Tests/Client/ClientProtocolCompatibilityTest.cs

```diff
@@ -93,32 +93,6 @@ public class ClientProtocolCompatibilityTest : ClientTestBase
             }
         }
 
-        /// <summary>
-        /// Tests that client can connect to old server nodes and negotiate common protocol version. 
-        /// </summary>
-        [Test]
-        public void TestClientNewerThanServerReconnectsOnServerVersion()
```

Comment:
We discussed this with Pavel and decided to remove the test because its logic is incorrect. But currently, it's already returned because it's not falling anymore.